adhocracy4/comments/static/comments/CommentBox.jsx

```diff
@@ -51,12 +51,16 @@ class CommentBox extends React.Component {
           })
         }
       })
-      .fail(respone => {
+      .fail(response => {
+        var errorMessage = JSON.parse(response.responseText).comment[0]
         if (typeof parentIndex !== 'undefined') {
-          this.updateStateComment(parentIndex, undefined, {replyError: true})
+          this.updateStateComment(
+            parentIndex,
+            undefined, {replyError: true, errorMessage: errorMessage})
         } else {
           this.setState({
-            error: true
+            error: true,
+            errorMessage: errorMessage
```

Comment:
the states `errorMessage` is not reset only the `error` flag. i think that's okay, but i am not sure what the most reactish way would be.

Reply:
I am not sure, if I understand what you mean. Do you mean I should set the errorMessage also if the submit does not fail? What value would you set it to? Undefined, empty String?